### ISSUE TITLE
Theme Handbook: Updating accessibility review request 

### ISSUE THREAD

**Issue by TeBenachi** [WordPress/Documentation-Issue-Tracker#98]
The [accessibility review process](https://make.wordpress.org/themes/handbook/review/accessibility/) has been changed and need to be updated. 

**[Suggested Draft]** 
**Requesting an accessibility review of a live theme:**
When an existing theme is updated with an `accessibility-ready` tag, the theme is sorted into the [accessibility queue](https://themes.trac.wordpress.org/query?status=closed&keywords=~accessibility-ready&changetime=90+days..&groupdesc=1&owner=themetracbot&report=29&col=id&col=summary&col=changetime&col=resolution&col=owner&col=reporter&desc=1&order=changetime) on the trac repository. Once the accessibility review is completed, the theme author will receive a message with the recommendations. If the theme author fails to make adequate corrections promptly, it could result in suspension from the directory.

**Comment by carolinan** [comment]
This draft looks correct to me, thank you.

Are there any other places where we might need to update this?

**Comment by TeBenachi** [comment]
@carolinan Thank you so much for checking!

**Comment by TeBenachi** [comment]
I think it is safe to remove this sentence from the How to be “Accessibility Ready” section. 

_**Themes with this tag may not be made live in the theme repository until they have passed an accessibility audit.**_
